### PR TITLE
Ros2 init config

### PR DESCRIPTION
**Pull request by hect95**
This PR ports the ROS1 version of [av_dbw_ros1](https://github.com/ipab-rad/av_dbw_ros1) to ROS 2 Humble
- Cyclone DDS ROS RMW is used + configured for high msg thorughput

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+RUN curl -sSL https://bitbucket.org/DataspeedInc/ros_binaries/raw/master/dataspeed.key -o /usr/share/keyrings/dataspeed-archive-keyring.gpg \
+    && sh -c 'echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/dataspeed-archive-keyring.gpg] http://packages.dataspeedinc.com/ros2/ubuntu \
+    $(. /etc/os-release && echo $UBUNTU_CODENAME) main" | sudo tee /etc/apt/sources.list.d/ros2-dataspeed-public.list > /dev/null'
+
+# Setup Dataspeed rosdep
+RUN sh -c "echo \"yaml http://packages.dataspeedinc.com/ros2/ros-public-\$ROS_DISTRO.yaml \$ROS_DISTRO\" \
+    > /etc/ros/rosdep/sources.list.d/30-dataspeed-public-\$ROS_DISTRO.list"
+
+
+# Install basic dev tools (And clean apt cache afterwards)
```
Comment (GreatAlexander) [inline Dockerfile]: Comment no longer accurate

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+RUN apt-get update \
+    && rosdep update \
+    && DEBIAN_FRONTEND=noninteractive \
+        apt-get -y --quiet --no-install-recommends install \
+        # Dataspeed Ford Drive-By-Wire
+        ros-"$ROS_DISTRO"-dbw-ford \
+        # Dataspeed Power Distribution System
+        # ros-noetic-dataspeed-pds-can \
```
Comment (GreatAlexander) [inline Dockerfile]: Does this work? Or leftover copy-paste?

Comment (hect95) [reply Dockerfile]: Left ofver-copy, I couldn't find the ROS 2 version of that pkg, last [commit](https://bitbucket.org/DataspeedInc/dataspeed_pds/src/master/) was over 4 years ago. Will remove it

Comment (hect95) [reply Dockerfile]: Done

File: Dockerfile (hunk 1 above)
Comment (hect95) [reply Dockerfile]: True, I have changed it to something mor accurate

**Review by GreatAlexander — APPROVED** [review]

**Pull request merged**